doc/cli/npm-adduser.md

```diff
@@ -66,6 +66,13 @@ different hostnames. See `always-auth` in `npm-config(7)` for more details on
 always-auth. Registry-specific configuration of `always-auth` takes precedence
 over any global configuration.
 
+### auth-type
+
+* Default: `'legacy'`
+* Type: `'legacy'`, `'sso'`, `'saml'`, `'oauth'`
+
+What authentication strategy to use with `adduser`/`login`.
```

Comment:
This description feels a little redundant given that this _is_ the `adduser`/`login` doc. Some description of under what circumstances you'd want to use the other strategies would probably be helpful, even if it's mostly a pointer to, like, npme docs.